src/components/input/Select.tsx

```diff
@@ -275,23 +275,27 @@ type ListboxCSSProps =
   | 'marginTop';
 
 /**
- * Manages the listbox position manually to make sure it renders "next" to the
+ * Manages the popover position manually to make sure it renders "next" to the
  * toggle button (below or over). This is mainly needed when the listbox is used
  * as a popover, as that makes it render in the top layer, making it impossible
  * to position it relative to the toggle button via regular CSS.
+ *
+ * @param asNativePopover - Native popover API is used to toggle the popover
+ * @param alignToRight - Whether the popover should be aligned to the right side
+ *                       of the button or not
  */
-function useListboxPositioning(
+function usePopoverPositioning(
```

Comment:
I removed all references to "listbox" from this hook, because it is likely going to be extracted and decoupled from that.